operations/experimental/add-ipv6-security-group.yml

```
@@ -0,0 +1,22 @@
+# --- Add IPv6 Security Groups ---
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/default_running_security_groups
+  value:
+  - public_networks
+  - dns
+  - ipv6_public_networks
+
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/default_staging_security_groups
+  value:
+  - public_networks
+  - dns
+  - ipv6_public_networks
+
+- type: replace
+  path: /instance_groups/name=api/jobs/name=cloud_controller_ng/properties/cc/security_group_definitions/-
+  value:
```

As discussed in a call, I would like to have a note _in_ the file that makes clear what you enable if you take this opsfile wholesale.
```suggestion
  value:
  # This is a sample IPv6 allowlist that allows the entirety of
  # the public IPv6 address space. Please note that this may include
  # the Diego cells and other nodes managed by BOSH, which might be
  # considered "internal" and are protected from external requests
  # via global firewall rules that apply for the CF foundation.
  # Please exclude any addresses from the range that should not be
  # accessible from app containers.
```